types/cypress-cdp/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "cypress": "9.7.0",
+        "devtools-protocol": "0.0.1173815"
```

Comment:
I don't think these should be pinned to exact versions; a carat range or even >= would be better.

Reply:
Hi jake! I used the same version as the cypress-cdp package. There is a renovate branch created by a bot that updates to Cypress 12.7.2 but I've not done extensive testing of all comestible versions. Nevertheless, I will update with >=, it might be the most suitable. Thanks!

Reply:
(You'd just have to change it again later if they change...)

Reply:
Okay, let's keep it that way then, thanks again for your feedback!